docs: tw713-json-rpc (#3311)
OffchainLabs/arbitrum-docs: docs/build-decentralized-apps/arbitrum-vs-ethereum/03-rpc-methods.mdx

| ----------- | -------------------------------------------------------------------------------------------------- |
| `requestId` | On parent to child chain transactions, this field is added to indicate position in the Inbox queue |

## Arbitrum specific `eth` JSON-RPC additions

> - `eth_sendRawTransactionConditional` submits a raw signed transaction with extra preconditions the sequencer must verify before accepting it. If any condition fails, the tx is rejected with JSON-RPC error -32003 (rejected).

Reviewer: ```suggestion
- `eth_sendRawTransactionConditional` submits a raw signed transaction with extra preconditions that the sequencer must verify before accepting it. If any condition fails, the transaction is rejected with JSON-RPC error -32003 (rejected).
```

| `ConditionalOptions` payload        | Description                                |
| ----------------------------------- | ------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |

> | `knownAccounts`                     | map of address → (storageRootHash          | { `slot: expectedValue` }). The tx is rejected unless each account's storage root matches, or each listed slot still holds the expected value. |

Reviewer: ```suggestion
| `knownAccounts`                     | map of address → (storageRootHash          | { `slot: expectedValue` }). The transaction is rejected unless each account's storage root matches, or each listed slot still holds the expected value. |
```

> | `blockNumberMin` / `blockNumberMax` | L1 block-number window the tx is valid in. |

Reviewer: ```suggestion
| `blockNumberMin` / `blockNumberMax` | L1 block-number window the transaction is valid. |
```
